source/tests/unit/ni_fake_service_tests.cpp

```diff
@@ -82,15 +100,15 @@ TEST(NiFakeServiceTests, NiFakeService_InitWithOptionsSucceeds_CreatesAndStoresS
   EXPECT_TRUE(status.ok());
   EXPECT_EQ(kDriverSuccess, response.status());
   nidevice_grpc::Session session = response.vi();
-  EXPECT_EQ(kTestViSession, session_repository.access_session(session.id(), ""));
```

Comment:
I weakened all these asserts from "I expect the session_repository to give me the driver-specific session id" to "I expect the session_repository to give *some valid server allocated session id*". 

This is also a case where I could add back more specific asserts on the contests of the session_**resource**_repository. But I think it's better to leave that out and focus on higher level testing in this class.

Reply:
This weakening of the asserts makes sense to me.

I agree, this high level of testing feels right here.